node/pkg/accountant/query_test.go

```diff
@@ -108,5 +108,19 @@ func TestParseBatchTransferStatusNotFoundResponse(t *testing.T) {
 }
 
 func TestParseBatchTransferStatusPendingResponse(t *testing.T) {
-	//TODO: Write this test once we get a sample response.
+	pendingEntryJson := []byte("{\"digest\":\"65hmAN4IbW9MBnSDzYmgoD/3ze+F8ik9NGeKR/vQ4J4=\",\"tx_hash\":\"CjHx8zExnr4JU8ewAu5/tXM6a5QyslKufGHZNSr0aE8=\",\"signatures\":\"1\",\"guardian_set_index\":0,\"emitter_chain\":2}")
+	var pendingEntryResponse TransferStatusPending
+	err := json.Unmarshal(pendingEntryJson, &pendingEntryResponse)
+	require.NoError(t, err)
+
+	pendingEntriesJson := []byte("[{\"digest\":\"65hmAN4IbW9MBnSDzYmgoD/3ze+F8ik9NGeKR/vQ4J4=\",\"tx_hash\":\"CjHx8zExnr4JU8ewAu5/tXM6a5QyslKufGHZNSr0aE8=\",\"signatures\":\"1\",\"guardian_set_index\":0,\"emitter_chain\":2}]")
+	var pendingEntriesResponse []TransferStatusPending
+	err = json.Unmarshal(pendingEntriesJson, &pendingEntriesResponse)
```

Comment:
pls test properly and compare against a preexisting struct, improper parsing would not necessarily lead to an error being returned.

Reply:
Oops! I was so excited when I finally got json parsing to work that I forgot to actually verify the results! Added that.